### PR TITLE
chore: bump to pg8.*

### LINKED ISSUE #24: pg tests fail on node 14
See build log here https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-js-contrib/20/workflows/86b033c2-7114-4093-8dbb-3bd537d206e1/jobs/119

cc/ @markwolff

### PR DESCRIPTION
**Pull request by markwolff**
## Which problem is this PR solving?

- Fixes #24 

## Short description of the changes

- Adds `8.*` to versions matrix. Only test on `pg@8.*`

### CODE REVIEW

File: plugins/node/opentelemetry-plugin-pg/package.json [hunk 1]
```diff
@@ -57,12 +57,13 @@
...
     "mocha": "^6.2.0",
     "nyc": "^15.0.0",
-    "pg": "^7.12.1",
+    "pg": "^8.0.3",
     "rimraf": "^3.0.0",
     "ts-mocha": "^6.0.0",
     "ts-node": "^8.6.2",
     "tslint-consistent-codestyle": "^1.16.0",
     "tslint-microsoft-contrib": "^6.2.0",
+    "cross-env": "^6.0.0",
```
Comment (vmarchaud) [inline plugins/node/opentelemetry-plugin-pg/package.json:66]: why do we need `cross-env` ? Because there are multiple versions ?

Comment (markwolff) [reply plugins/node/opentelemetry-plugin-pg/package.json:66]: This package is using it to run the tests locally and I saw it was missing. Are we relying on certain dev dependencies being hoisted via lerna?

The version does not matter to this package.

Comment (dyladan) [reply plugins/node/opentelemetry-plugin-pg/package.json:66]: It's being used on line 11. We probably have just gotten lucky that it is also in the dependency tree somewhere else and was installed. We do not use hoist.

**Comment by dyladan** [comment]
@mayurkale22 update branch button not working for me here. Is it working for you? @markwolff may need you to manually merge master back something is weird.

**Comment by mayurkale22** [comment]
> @mayurkale22 update branch button not working for me here. Is it working for you? @markwolff may need you to manually merge master back something is weird.

Same problem for me.

**Comment by dyladan** [comment]
@mayurkale22 I think it may be because of the history mangling we had to do in order to maintain the file change history. It might be worth it to blow away the history...

**Pull request merged**